exec/java-exec/src/main/resources/rest/static/js/ace-code-editor/snippets/sql.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Drill SQL Syntax Snippets
+ */
+
+ace.define("ace/snippets/sql",["require","exports","module"], function(require, exports, module) {
+"use strict";
+
+exports.snippetText = "snippet info\n\
+	select * from INFORMATION_SCHEMA.${1:<tableName>};\n\
+snippet sysmem\n\
+	select * from sys.mem;\n\
+snippet sysopt\n\
+	select * from sys.opt;\n\
+snippet sysbit\n\
+	select * from sys.bit;\n\
+snippet sysconn\n\
+	select * from sys.conn;\n\
+snippet sysprof\n\
+	select * from sys.prof;\n\
+snippet cview\n\
+	create view ${1:[workspace]}.${2:<viewName>} ( ${3:<columnName>} )  as \n\
+	${4:<query>};\n\
+snippet ctas\n\
+	create table ${1:<tableName>} ( ${2:<columnName>} )  as \n\
+	${3:<query>};\n\
+snippet ctemp\n\
+	create temporary table ${1:<tableName>} ( ${2:<columnName>} )  as \n\
```

Comment:
Could you please also add snippet for create function: `CREATE FUNCTION USING JAR '<jar_name>.jar';`?

Reply:
Done.